src/main/java/com/blackducksoftware/integration/fortify/batch/util/MappingParser.java

```diff
@@ -62,8 +70,16 @@
 
     private final static String Q_connector = "+and+";
 
+    private final static String DYNAMIC_SCAN_REQUEST = "DYNAMIC_SCAN_REQUEST";
+
     private final static Logger logger = Logger.getLogger(MappingParser.class);
 
+    private final SpringConfiguration springConfiguration;
```

Comment:
Closer.. but nothing should take a SpringConfiguration, it should take the beans made from the SpringConfiguration. So for instance, this constructor should take: 

- an instance of FortiyApplicationVersionApi